dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -201,6 +217,29 @@ public void patch(Context context, HttpServletRequest request, String apiCategor
                 throw new ResourceNotFoundException(apiCategory + "." + model + " with id: " + uuid + " not found");
             }
             List<Operation> operations = patch.getOperations();
+
+            /**
+             * Although users can update their own password, other
+             * patch operations are for administrators only. Throwing
+             * the error from here.
+             */
+            boolean isAdmin;
+            try {
+                isAdmin = authorizeService.isAdmin(context);
+            } catch (SQLException e) {
+                throw new RuntimeException(e.getMessage(), e);
+            }
+            for (Operation op: operations) {
+                if (op.getPath().contentEquals("/canLogin")
+                        || op.getPath().contentEquals("/certificate")
+                        || op.getPath().contentEquals("/netid")
+                        || op.getPath().contentEquals("/email")) {
```

Comment:
Shouldn't it be possible for a user to change his e-mail address?

Reply:
this will be a new feature compared to what we have in the current dspace.
There are also some additional complexity related to the change of the email such as
- the risk that a spammer will register in the system using a valid email and after will change it email (of course we will have log to track that)
- the risk to put a wrong email that will not go through the same validation procedure

I recommend to do not include such feature in this PR but eventually open a JIRA ticket to discuss it

Reply:
Since this will be a new feature that probably needs more analysis, I agree with the idea of a new JIRA ticket.

I also agree that the user should be able to change their email address.